app.js

```diff
@@ -22,53 +22,54 @@ const portfolios = [
     seelive: 'See live',
     seesource: 'See Source',
   },
 
   {
     title: 'Tonic',
     listItems: ['CANOPY', 'Back End Dev', '2015'],
-    image: './images/work3.png',
-    content: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    image: './images/project3_desktop.png',
+    content: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
     seelive: 'See live',
     seesource: 'See Source',
   },
 
   {
     title: 'Multi-Post Stories',
     listItems: ['CANOPY', 'Back End Dev', '2015'],
-    image: './images/work4.png',
-    content: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    image: './images/project4_desktop.png',
+    content: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
     seelive: 'See live',
     seesource: 'See Source',
```

Comment:
- [ ] [Optional] I would suggest you please use the blur background so the image and text don't overlap to each other and if you see the design then you will find blur background beside the popup window
### Your design
![his](https://user-images.githubusercontent.com/81727988/216746695-c5f25871-f4ad-43f7-93bc-50a8370f1283.png)


### Figma design 

![figma](https://user-images.githubusercontent.com/81727988/216746698-070b7fa3-829b-48a9-a391-2da57ceed9cb.png)

Reply:
@roshan-bajgain thanks, I'll fix that during the weekend!